app/code/Magento/Checkout/view/frontend/web/template/shipping-information/address-renderer/default.html

```diff
@@ -8,7 +8,7 @@
     <!-- ko text: address().prefix --><!-- /ko --> <!-- ko text: address().firstname --><!-- /ko --> <!-- ko text: address().middlename --><!-- /ko -->
     <!-- ko text: address().lastname --><!-- /ko --> <!-- ko text: address().suffix --><!-- /ko --><br/>
     <!-- ko text: address().street --><!-- /ko --><br/>
-    <!-- ko text: address().city --><!-- /ko -->, <span data-bind="html: address().region"></span> <!-- ko text: address().postcode --><!-- /ko --><br/>
+    <!-- ko text: address().city --><!-- /ko --><!-- ko if: !(address().region.trim() == "") -->, <span data-bind="html: address().region"></span><!-- /ko --> <!-- ko text: address().postcode --><!-- /ko --><br/>
```

Comment:
Use strict equality or check on string length.

Reply:
we will fix this on our side. thank you for review